android/sdk/src/main/java/com/taobao/weex/ui/module/WXDomModule.java

```diff
@@ -118,6 +123,16 @@ public Object callDomMethod(String method, JSONArray args, long... parseNanos) {
           }
           new UpdateComponentDataAction(mWXSDKInstance, args.getString(0), JSONUtils.toJSON(args.get(1)), args.getString(2)).executeAction();
           break;
+        case BATCH_BEGIN: {
+          String ref = args.size() >= 1 ? args.getString(0) : null;
```

Comment:
args may equals null

Reply:
@Darin726   thanks, I have checked value for args, just return if null